### DISCUSSION TITLE
Caching in CI/CD environments?

### DISCUSSION THREAD

**Discussion by rcowsill** [containers/podman#10091] [Q&A]
Many CI/CD systems run builds in a fresh environment every time. That means there are no cached layers present, so container image builds start from scratch each time.

Docker has a few ways to inject layers into the build cache: the `--cache-from` option and `docker save`/`docker load`. These make it possible to export docker's build cache into the CI/CD system's cache and then restore it for later builds.

Is there any way to do that in podman? It seems that `--cache-from` is a no-op and `podman load` doesn't add the loaded layers to the cache.

**Comment by mheon** [comment]
@nalind @TomSweeneyRedHat This looks like a Buildah question?
